packages/experimental/src/components/APIKeyModal/APIKeyModal.stories.js

```diff
@@ -48,8 +48,33 @@ const defaultProps = {
   successHeader: 'API key successully created',
 };
 
-const Template = (args) => {
-  return <APIKeyModal {...args} />;
+const timeout = () => new Promise((resolve) => setTimeout(resolve, 2000));
+
+const MinimalTemplate = (args) => {
+  const [open, setOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const onCloseHandler = () => {
+    setOpen(false);
+  };
```

Comment:
A function with just 1 hook call looka redunant to me and you can achieve the same effect with an inline annonymus func...